core/io/resource.h

```diff
@@ -81,13 +81,14 @@ class Resource : public RefCounted {
 	void _take_over_path(const String &p_path);
 
 public:
-	static Node *(*_get_local_scene_func)(); //used by editor
-	static void (*_update_configuration_warning)(); //used by editor
+	static Node *(*_get_local_scene_func)(); //Used by editor.
+	static void (*_update_configuration_warning)(); //Used by editor.
 
 	void update_configuration_warning();
 	virtual bool editor_can_reload_from_file();
-	virtual void reset_state(); //for resources that use variable amount of properties, either via _validate_property or _get_property_list, this function needs to be implemented to correctly clear state
-	virtual Error copy_from(const Ref<Resource> &p_resource);
+	virtual void reset_state(); //For resources that use variable amount of properties, either via _validate_property or _get_property_list, this function needs to be implemented to correctly clear state.
+	static Ref<Resource> get_remap_resource(const Ref<Resource> &p_resource, HashMap<Ref<Resource>, Ref<Resource>> p_remap_cache, const Ref<Resource> p_fallback, Node *p_for_scene);
```

Comment:
This method shouldn't be mixed in here. You could put it under the static methods above.